confignode/src/main/java/org/apache/iotdb/confignode/manager/ProcedureManager.java

```diff
@@ -120,6 +122,18 @@ public void addConfigNode(TConfigNodeRegisterReq req) {
     this.executor.submitProcedure(addConfigNodeProcedure);
   }
 
+  /**
+   * generate a procedure, and execute remove confignodeby one by one
+   *
+   * @param removeConfigNodePlan remove config node plan
+   */
+  public void removeConfigNode(RemoveConfigNodePlan removeConfigNodePlan) {
+    RemoveConfigNodeProcedure addConfigNodeProcedure =
```

Comment:
variable naming, `addConfigNodeProcedure` to `removeConfigNodeProcedure`

Reply:
fixed